src/Util/OlStyleUtil.ts

```diff
@@ -442,6 +472,27 @@ class OlStyleUtil {
     switch (func.name) {
       case 'property':
         return feature?.get(args[0] as string);
+      case 'case':
+        type FCaseParameter = {
+          case: Expression<boolean>;
+          value: Expression<PropertyType>;
+        };
+        const caseArgs: Fcase['args'] = args as Fcase['args'];
+        let match;
+        for (let index = 0; index < caseArgs.length; index++) {
+          const caseArg = caseArgs[index] as FCaseParameter;
+          if (index === caseArgs.length - 1) {
+            match = caseArg;
+            break;
+          } else if (caseArg.case === true) {
+            match = caseArg.value;
+            break;
```

Comment:
This branches seem a little unintuitive, maybe some comments or a test for these branches might help?